Corgibytes.Freshli.Cli/Functionality/Cache.cs

```diff
@@ -4,89 +4,89 @@
 using System.IO;
 using System.Linq;
 using System.Runtime.Serialization;
+using Corgibytes.Freshli.Cli.Resources;
 using Microsoft.EntityFrameworkCore;
 
-namespace Corgibytes.Freshli.Cli.Functionality
+namespace Corgibytes.Freshli.Cli.Functionality;
+
+[Serializable]
```

Comment:
I like having the single blank line between the `namespace` and the `class` definition. I think it makes sense to apply that to the other files as well.